README.md

```diff
@@ -143,7 +144,7 @@ grunt watch
 npm link
 
 cd Foo
-npm install --save git://github.com/turn/Smart-Table.git#v0.3.8
+npm install --save git://github.com/turn/Smart-Table.git#v0.3.12
```

Comment:
The README.md file is generated by the README.tmpl.md file, which was programmed to display the exact version, I suppose.